app/controllers/use_case_controller.rb

```diff
@@ -66,6 +60,18 @@ def attribute_changes?
     !@model_instance.changed_attributes.blank?
   end
 
+  def build_model_instance
+    if params[:action] == 'new'
+      filtered_params = {}
+    else
+      filtered_params = self.permitted_params
+    end
+    @model_instance = self.model_class.new(filtered_params)
+
+    # TODO unify with how the factories create model instances
+    @model_instance.build_proposal(flow: 'linear', requester: current_user)
+  end
+
   def find_model_instance
     @model_instance ||= self.model_class.find(params[:id])
```

Comment:
without knowing much about this controller, my feedback is that all of this `model_class` and `model_instance` stuff is kinda confusing. Don't currently have a suggestion on how to change it because I am not exactly sure on why it is here to begin with, but thought I'd provide that feedback as a semi-new person to this codebase who is seeing things the way that other new people might see it.